### ISSUE TITLE
Feature Request: Highlight Cursor

### ISSUE THREAD

**Issue by Jiminimonka** [MaartenBaert/ssr#866]
Please add an option to highlight the cursor when recording - either make it bigger or put a circle around it (some software uses a yellow circle around the mouse cursor) so its easier for people watching the video to see the cursor.
Thanks.

**Comment by boban-dj** [comment]
Yes please, an options for yellow circle around the mouse cursor
